README.md

```diff
@@ -983,6 +984,105 @@ This section describes code style for [ECMAScript 2015 Language Specification](h
 
 [⬆ back to TOC](#table-of-contents)
 
+### Arrow functions
+
+* [Arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+  should be used where an anonymous function is expected (when you need function and you don't want bind it to an
+  identifier):
+
+  > Explanation:
+  > * `this` keyword in arrow function binds to the upper scope's `this`. It prevents usage invalid `this` or usage less
```

Comment:
> It prevents misusing the this keyword inside a callback.

I think `misusing` is not very good word: `this` value may be used inside arrow function.
Main idea here that arrow functions lexically binds the `this` value. In most cases this is expected behavior. 

> Also it replaces the less efficient method for binding context - the Function.prototype.bind method.

Technically `this != context`.